app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,340 @@
+:root {
+  --size-border: 2px solid var(--text-color);
+  --box-border: 2px solid var(--shadow-color);
+  --prmary-color: #f4f4f4;
+  --bacground-color: #eee;
+  --main-color: #3778c2;
+  --secondary-color: #5fb523;
+  --text-color: #434b54;
+  --bg-color: #f5f6f8;
+  --on-main: #fff;
+  --on-secondary: #fff;
+  --shadow-color: #f1f1f2;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  height: 100vh;
+  width: 100%;
+  background-color: var(--bacground-color);
+}
+
+.new-edit-cat {
+  cursor: pointer;
+  border: none;
+  background: inherit;
+  color: white;
+  font-size: 1.3rem;
+  margin-right: 10px;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.flex-center {
+  align-items: center;
+  justify-content: space-between;
+}
+
+main {
+  max-width: 540px;
+  width: 98%;
+  height: auto;
+  min-height: 95vh;
+  margin: 1rem auto;
+  position: relative;
+
+  /* border: 1px solid rgba(134, 134, 134, 0.507); */
+}
+
+button a {
+  color: var(--on-secondary);
+  font-size: 1.3rem;
+}
+
+form {
+  margin: 0 auto;
+  padding: 1rem;
+  background-color: var(--bg-color);
+  border-radius: 4px;
+}
+
+/* SPLASH PAGE */
+
+.splash-page {
+  width: 100%;
+  height: 90vh;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  background-color: var(--main-color);
+}
+
+.splash-page h1 {
+  font-size: 2rem;
+  padding: 0.3rem 1rem;
+  color: var(--secondary-color);
+  border: var(--box-border);
+  border-radius: 8px;
+  font-weight: 900;
+}
+
+.buttons {
+  width: 90%;
+}
+
+.buttons form {
+  width: 100%;
+  background-color: var(--main-color);
+  margin: 0 auto;
+}
+
+.buttons button {
+  width: 100%;
+  margin: 0 auto;
+  background-color: #5fb523;
+  padding: 0.5rem;
+  border: var(--bg-color);
+  color: var(--on-secondary);
+  border-radius: 4px;
+  font-weight: 700;
+  font-size: 1.2rem;
+}
+
+/* LOG IN and SIGN UP */
+
+.nav-splash {
+  width: 100%;
+  background-color: var(--shadow-color);
+  color: var(--main-color);
+  padding: 0.5rem;
+}
+
+.icon {
+  width: 10%;
+}
+
+.icon i {
+  margin-left: 50%;
+  padding: 0.3rem 0.5rem;
+  text-align: right;
+  font-size: 1.2rem;
+  color: var(--main-color);
+}
+
+.nav-splash h1 {
+  color: var(--main-color);
+}
+
+.actions {
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.actions input {
+  font-weight: 700;
+  border: var(--size-border);
+  border-radius: 4px;
+  padding: 0.2rem 1rem;
+}
+
+.act a {
+  font-weight: 700;
+  border: var(--size-border);
+  border-radius: 4px;
+  padding: 0.2rem 1rem;
+}
+
+/* CATEGORIES PAGE */
+
+/* .home-header-categories h1 {
+  width: 100%;
+  line-height: 70px;
+  text-align: center;
+  font-size: 24px;
+  background-color: var(--main-color);
+  color: var(--on-secondary);
+} */
+
```

Comment:
- [ ] Kindly remove the commented code.

Reply:
Yeah sure 😢 I've forgotten to remove it!!